lib/generators/rolemodel/modals/templates/app/views/layouts/application.html.slim

```diff
@@ -7,13 +7,10 @@ html
     = stylesheet_pack_tag 'stylesheets', 'data-turbolinks-track': 'reload'
     = javascript_pack_tag 'application', 'data-turbolinks-track': 'reload'
   body.flex
+    = turbo_frame_tag 'modal'
```

Comment:
Rather than doing a full file replace. Should just find the body and insert these lines